File: 03/01/python/plugins/parrot.py
class Parrot:
  def __init__(self, name):
    self.name = name
  def name(self):
    return self.name
  def greeting(self):
    return "parrot greeting"
  def menu(self):
    return "parrot menu"


def create(name):
  return Parrot(name)

File: 03/01/java/GenerickaTvornica/src/hr/fer/zemris/ooup/lab3/model/plugins/Parrot.java
package hr.fer.zemris.ooup.lab3.model.plugins;

import hr.fer.zemris.ooup.lab3.model.Animal;

public class Parrot extends Animal {
    private String animalName;

    public Parrot(String name) {
        this.animalName = name;
    }

    @Override
    public String name() {
        return this.animalName;
    }

    @Override
    public String greet() {
        return "parrot greet";
    }

    @Override
    public String menu() {
        return "parrot menu";
    }
}
